src/lib/classes/notification.js

```diff
@@ -479,6 +479,11 @@ export class Notification {
 
             notificationData[field.label] = fieldQuad ? fieldQuad.object.value : null;
           }
+
+          // Take the actor webid and construct an object, fetching name and profile image
+          const actor = notificationData.actor && (await getBasicPod(notificationData.actor));
+          notificationData = { ...notificationData, actor };
```

Comment:
If actor is `undefined` (what's tested on line 484), do we want to add it to notificationData, or could this be something like
```
if(notificationData.actor) {
  notificationData = { ...notificationData, await getBasicPod(notificationData.actor)}
}
```
?